### PR TITLE
Track editing and deleting messages

### PR DESCRIPTION
**Pull request by basine**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@ Carthage/
 Wire-iOS/ComponentsVersions.plist
 /Libraries
 /ConfigurationSets/Private
+/ConfigurationSets/Development_private.xcconfig
```
Comment (mikeger) [inline .gitignore]: I guess this is also not necessary for the moment

**Comment by mikeger** [comment]
@basine weird, seems no UI changes but some tests are failing @daehn do we have tests failing on `release/2.14`?

**Comment by marcoconti83** [comment]
@daehn any comment on this?

**Comment by daehn** [comment]
I needed to add an exception for the release branch to build on pushes on Travis, which it now does. But https://github.com/wireapp/wire-ios/pull/106 which is branched of the release branch succeeds.

**Comment by mikeger** [comment]
@marcoconti83 @basine what about redundant gitignore changes?

File: .gitignore (hunk 1 above)
Comment (marcoconti83) [inline .gitignore]: This was added because we do not want to commit this by accident

**Comment by marcoconti83** [comment]
@mikeger that ignore was done because we want to avoid committing that file accidentally after a session of debugging

**Comment by mikeger** [comment]
@marcoconti83 we cannot commit it by accident because the import declaration for this file is looking on the different location.

**Pull request merged**